src/briefcase/commands/dev.py

```diff
@@ -75,8 +76,23 @@ def add_options(self, parser):
             help="Run the app in test mode",
         )
 
+    def install_app_code(
+        self, app: AppConfig, venv: VenvContext, dist_info_path: str, **options
+    ):
+        """This method creates dist-info metadata so Python recognizes the app as an
+        installed package without requiring pip install. For desktop platforms, source
+        code is accessible via PYTHONPATH at runtime.
+
+        :param app: The config object for the app
+        :param venv: The context object used to run commands inside the virtual
+            environment.
+        :param dist_info_path: The path to the .dist-info directory for the app
```

Comment:
dist_info_path shouldn't be needed as a separate argument; it can be entirely derived from `app`. This likely indicates that the evaluation of `dist_info_path` needs to be turned into a property.